src/data/search.tsx

```diff
@@ -76,5 +96,25 @@ export function searchContent(needle: string, contentType: string) {
     "rules": searchableRuleProvisions,
   }[contentType];
 
-  return data.filter((x: IndexedObject) => x.content.includes(needle));
+  switch (contentType) {
+    default:
+      const searchResult = searchableProvisions.search(needle);
+      console.log('deault SEARCH RESULTS: ', searchResult);
+      return searchResult;
+
+    case "constitution":
+      const searchResultC = searchableProvisions.search(`${needle}*`);
+      console.log('CONSTITUTION SEARCH RESULTS: ', searchResultC);
+      return searchResultC;
```

Comment:
The value returned from lunr is not the object indexed, it is search result information. Using the `ref` field in the search result, you must look up the matched objects (this needs to be stored outside of lunr) and return those, in the same format that the old search method returned: a JS object with `id`, `title` and `item` fields.

Reply:
If you've attached the search data as described above, you could do this:

```
return searchResultC.map(m => searchableProvisions._searchData[m.ref]);
```